docs/dapps/getting-started.md

```diff
@@ -8,91 +8,531 @@ import UntranslatedPageText from "@site/src/components/UntranslatedPageText";
+import { PrivateKeyWallet} from '@alephium/web3-wallet'
+import configuration from '../alephium.config'
+import { TokenFaucet, Withdraw } from '../artifacts/ts'
+
+async function withdraw() {
+
+  //Select our network defined in alephium.config.ts
+  const network = configuration.networks[configuration.defaultNetwork]
+
+  //NodeProvider is an abstraction of a connection to the Alephium network
+  const nodeProvider = new NodeProvider(network.nodeUrl)
+
+  //Sometimes, it's convenient to setup a global NodeProvider for your project:
+  web3.setCurrentNodeProvider(nodeProvider)
+
+  //Connect our wallet, typically in a real application you would connect your web-extension or desktop wallet
+  const wallet = new PrivateKeyWallet({privateKey: '672c8292041176c9056bb0dd1d91d34711ceed2493b5afc83f2012b27df2c559' })
+
+  // Compile the contracts of the project if they are not compiled
+  Project.build()
+
+  //.deployments contains the info of our `TokenFaucet` deployement, as we need to now the contractId and address
+  //This was auto-generated with the `cli deploy` of our `scripts/0_deploy_faucet.ts`
+  const deployments = await Deployments.from('.deployments.devnet.json')
+
+  //Make sure it match your address group
+  const accountGroup = 1
+
+  const deployed = deployments.getDeployedContractResult(accountGroup, 'TokenFaucet')
+
+  if(deployed !== undefined) {
+    const tokenId = deployed.contractId
+    const tokenAddress = deployed.contractAddress
+
+    // Submit a transaction to use the transaction script
+    // It uses our `wallet` to sing the transaction.
+    await Withdraw.execute(wallet, {
+      initialFields: { token: tokenId, amount: 1n }
+    })
+
+    // Fetch the latest state of the token contract, `mut balance` should have change
+    const faucet = TokenFaucet.at(tokenAddress)
+    const state = await faucet.fetchState()
+    console.log(state.fields)
+
+    // Fetch wallet balance see if token is there
+    const balance = await wallet.nodeProvider.addresses.getAddressesAddressBalance(wallet.account.address)
+    console.log(balance)
+  } else {
+    console.log('`deployed` is undefined')
+  }
+}
+
+// Let's perform one withdraw
+withdraw()
+```
+
+For the attentive people, you'll see something new coming from our `artifacts`: [Withdraw](https://github.com/alephium/nextjs-template/blob/main/contracts/withdraw.ral) which is a [TxScript](https://wiki.alephium.org/ralph/getting-started#txscript) required to interact with the `TokenFaucet` contract. Its code is quite simple, you can add this to you `contracts/withdraw.ral`
+
+```rust
+TxScript Withdraw(token: TokenFaucet, amount: U256) {
+    token.withdraw(amount)
+}
+```
+
+We now need to recompile our contracts to get the artifact for `Withdraw`
+
+```sh
+npx @alephium/cli@latest compile
+```
+
+The code of `token.ts` should be self-explanatory, you can read more on our [web3-sdk here](https://wiki.alephium.org/dapps/alephium-web3)
```

Comment:
Hm, I don't see the value of this sentence. Also, the wrong link is being used (`wiki` instead of `docs`). Lastly, as a reader, I would expect to land at a section that explains the code of `token.ts`, but instead I get to an page about the SDK.